### ISSUE TITLE
Allow references without .value

### ISSUE THREAD

**Issue by markacianfrani** [amzn/style-dictionary#721]
Currently, for outputReferences to work, you need to define your reference using .value like:
```
      "base"  : { "value": "{size.font.medium.value}" }
```

If your reference doesn't include .value it won't work:
```
    "base"  : { "value": "{size.font.medium}" }
```

The [design tokens working group first draft](https://design-tokens.github.io/community-group/format/#aliases-references) actually has the reference defined without .value
```
{
  "group name": {
    "token name": {
      "value": 1234
    }
  },
  "alias name": {
    "value": "{group name.token name}"
  }
}
```

Would you be open to supporting references without .value to better align with the spec's working draft? 

From what I can tell, we'd just need to modify around line 91 of https://github.com/amzn/style-dictionary/blob/main/lib/common/formatHelpers/createPropertyFormatter.js to maybe check if it's object and if the object has a value property, use that instead. 

Would be happy to put in a PR if you think that's a good approach.

**Comment by phantasmagoriadigital** [comment]
+1 This would be really helpful to synchronise our tokens between figma tokens and style dictionary. That way we can keep all our tokens as per W3C spec.

**Comment by esthercheran** [comment]
+1 Would be really helpful for us too.

**Comment by zetareticoli** [comment]
+1 Would be really useful to stay aligned with Figma plugin and Style Dictionary output

**Comment by chazzmoney** [comment]
Closed via #746
